### PR TITLE
W-12733611: Implement the tracing of the connection obtention

### PR DESCRIPTION
**Pull request by IvanAndresFritzler**

### CODE REVIEW

File: modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/ExtensionConnectionSupplier.java [hunk 1]
```diff
@@ -41,9 +47,14 @@
  */
 public class ExtensionConnectionSupplier {
 
+  public static final String MULE_GET_CONNECTION = "mule:get-connection";
   @Inject
```
Comment (fsgonz) [inline modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/ExtensionConnectionSupplier.java:51]: add a space.

File: modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/ExtensionConnectionSupplier.java [hunk 2]
```diff
@@ -117,4 +147,55 @@ public void setMuleContext(MuleContext muleContext) {
     this.lazyConnections =
         parseBoolean(muleContext.getDeploymentProperties().getProperty(MULE_LAZY_CONNECTIONS_DEPLOYMENT_PROPERTY, "false"));
   }
+
+  @Inject
+  public void setProfilingService(InternalProfilingService profilingService) {
```
Comment (fsgonz) [inline modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/ExtensionConnectionSupplier.java:152]: Isn't it possible to inject the coreEventTracer directly?

File: tests/infrastructure/src/main/java/org/mule/test/infrastructure/profiling/tracing/ExceptionEventMatcher.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
+package org.mule.test.infrastructure.profiling.tracing;
+
+import org.hamcrest.Description;
```
Comment (fsgonz) [inline tests/infrastructure/src/main/java/org/mule/test/infrastructure/profiling/tracing/ExceptionEventMatcher.java:3]: import order.

**Review by fsgonz — CHANGES_REQUESTED** [review]

File: modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/ExtensionConnectionSupplier.java [hunk 4]
```diff
@@ -41,9 +46,17 @@
  */
 public class ExtensionConnectionSupplier {
 
+  public static final String MULE_GET_CONNECTION = "mule:get-connection";
```
Comment (fsgonz) [inline modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/ExtensionConnectionSupplier.java:49]: shouldn't this be a connection-create for the levels to take it accordingly?

**Review by fsgonz — CHANGES_REQUESTED** [review]

**Review by sofiamorseletto — DISMISSED** [review]

**Review by fsgonz — APPROVED** [review]

**Comment by IvanAndresFritzler** [comment]
--validate

**Review by sofiamorseletto — APPROVED** [review]

**Review by fsgonz — APPROVED** [review]

**Review by fsgonz — APPROVED** [review]

**Comment by IvanAndresFritzler** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Pull request merged**